registry/src/App.js

```diff
@@ -32,6 +33,10 @@ function App() {
           path="/account/reset-password/:uuid"
           element={<ResetPassword />}
         />
+        <Route
```

Comment:
Ok, I see you have the route registered for verification using `uuid`.
But, here is the problem https://registry-phi.vercel.app/account/verify/uuid.

It still opens the page instead of 404 message when I use above url.

Reply:
@arteevraina , we would be sending the email to the user with this link `https://registry-phi.vercel.app/account/verify/<uuid>` , thus when you try to do it with `https://registry-phi.vercel.app/account/verify/uuid` it will redirect to 404.